assets/data-port/export/export-select-content-page.js

```diff
@@ -1,20 +1,24 @@
 import { Button, CheckboxControl } from '@wordpress/components';
-import { useReducer } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { useMergeReducer } from '../../react-hooks/use-merge-reducer';
 import { postTypeLabels } from '../../shared/helpers/labels';
 
 export const ExportSelectContentPage = ( { onSubmit } ) => {
-	const [ values, setValue ] = useReducer(
-		( state, newValues ) => ( { ...state, ...newValues } ),
-		{
-			course: false,
-			lesson: false,
-			question: false,
-		}
-	);
+	const [ values, updateValues ] = useMergeReducer( {
+		course: false,
+		lesson: false,
+		question: false,
+	} );
 	const submit = ( event ) => {
 		event.preventDefault();
-		onSubmit( values );
+		const selectedTypes = Object.entries( values ).reduce(
```

Comment:
Only a suggestion, so feel free to decide what's better. :)
Do you think it could be more readable with something like that? `Object.entries(values).filter( i => i.value ).map( i => i.type )`

Reply:
I kept the reduce so it only goes through the array once, but moved it to a helper, since this kind of indexed map to list of keys tends to come up often. 1ad9b61

Reply:
Not critical, but WDYT to write a test for it? Maybe it helps to document the purpose too. :)